app/src/main/java/com/example/myapplication/MainActivity.java

```diff
@@ -1,13 +1,44 @@
 package com.example.myapplication;
 
+
+import android.content.Intent;
 import android.support.v7.app.AppCompatActivity;
 import android.os.Bundle;
+import android.view.View;
+import android.widget.Button;
+import android.widget.Toast;
+
+
+public class MainActivity extends AppCompatActivity implements View.OnClickListener {
 
-public class MainActivity extends AppCompatActivity {
+    private  Button nhan,dong;
 
     @Override
     protected void onCreate(Bundle savedInstanceState) {
         super.onCreate(savedInstanceState);
         setContentView(R.layout.activity_main);
+        nhan= (Button) findViewById(R.id.nhan);
+        dong= (Button) findViewById(R.id.Dong);
+        nhan.setOnClickListener(this);
+        dong.setOnClickListener(this);
+    }
+
+
+    @Override
+    public void onClick(View v) {
+               switch (v.getId()){
```

Comment:
1 tab = 4 spaces
ở đây là 12 space r
```suggestion
    switch (v.getId()){
```